src/plugin/arc/layer/arclayerdescriptor.js

```diff
@@ -554,3 +561,38 @@ plugin.arc.layer.ArcLayerDescriptor.prototype.getNodeUI = function() {
 plugin.arc.layer.ArcLayerDescriptor.prototype.getFilterableTypes = function() {
   return [this.getId() + os.ui.data.BaseProvider.ID_DELIMITER + 'features'];
 };
+
+/**
+ * @inheritDoc
+ */
+plugin.arc.layer.ArcLayerDescriptor.prototype.getUrlKey = function() {
+  return this.url_ + '!!' + this.getTitle();
```

Comment:
Kevin is correct. This should `return this.getFilterKey();`

This is silly and we should consolidate those, but I'll write that up separately.